src/tallies/tally.cpp

```diff
@@ -250,6 +254,42 @@ Tally::Tally(pugi::xml_node node)
     }
   }
 
+  // Currently the pulse-height tally is only working in a selected type of
+  // settings Here these settings are checked again
+  bool exists = std::find(std::begin(scores_), std::end(scores_), 1001) !=
```

Comment:
Don't use 1001. Use the score enum you had defined earlier. That way no magic numbers appear in the code :smile:

Reply:
- I replaced 1001 with -27 since this is the correct macroscopic reaction number (https://serpent.vtt.fi/mediawiki/index.php/ENDF_reaction_MT%27s_and_macroscopic_reaction_numbers)
- I replaced 1001 with `ReactionType::PHT`. Anyway, this entire part will probably be removed in the next iteration.

Reply:
In the commit 16c1bdd, the pulse-height tally got the TallyScore=-17. It's not a ReactionType anymore. I think this issue is resolved by this.